IdentityCore/src/webview/systemWebview/ios/MSIDSFAuthenticationSessionHandler.h

```diff
@@ -32,7 +32,8 @@ API_AVAILABLE(ios(11.0))
 @interface MSIDSFAuthenticationSessionHandler : NSObject <MSIDWebviewInteracting>
 
 - (instancetype)initWithStartURL:(NSURL *)startURL
-                  callbackScheme:(NSString *)callbackURLScheme;
+                  callbackScheme:(NSString *)callbackURLScheme
+                         context:(id<MSIDRequestContext>)context;
```

Comment:
Looks like a good change to me, but I believe we'll have more places where context is not propagated. Is there an opportunity to simplify and ensure we always have correlationID in errors even if not propagated in all code paths?

Reply:
I found a common place(s) in public client where we have the context within the completion block. It is a ideal place but we like more discussion tomorrow as well